mev-boost-rs/src/relay_mux.rs

```diff
@@ -129,7 +134,7 @@ impl BlindedBlockProvider for RelayMux {
         let best_indices = select_best_bids(bids.iter().map(|(bid, i)| (bid.value(), *i)));
 
         if best_indices.is_empty() {
-            return Err(Error::NoBids)
+            return Err(Error::NoBids);
```

Comment:
is from the nightly cargo fmt?

try `just fmt` from the repo root dir to see if it stays